modules/features/account/src/main/res/layout-car/fragment_change_email.xml

```diff
@@ -1,147 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
-<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Have we checked that the screen is not displayed in the car? I belive they're not but can you please confirm by checking on Automotive app and Android Auto [DHU](https://developer.android.com/training/cars/testing/dhu#running-dhu) since the layouts were available earlier?

Reply:
Yes, I did test on Automotive App and Android Auto and that screen was not being used, but I'll just do another round to confirm.